### PR TITLE
fix(DB/Gossip): Fix Champion Cyssa Dawnrose gossip in Undercity

### PR DESCRIPTION
**Pull request by Asthalor**
Champion Cyssa had  Farseer Umbrua`s gossip. That happened because the gossip menu id had 2 gossip lines.
One the draenei and other the correct one.
I deleted the correct one and added to a new gossip id.
After that i updated the character to use the right one.

## Changes Proposed:
-  Deleted the line from the gossip menu id
-  Added the right line to a new gossip line
-  Updated the character to she uses the right line now

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6971
- Closes https://github.com/chromiecraft/chromiecraft/issues/1212

## SOURCE:

Picture of the fix:

image

Just to be sure i didnt broke the Farseer gossip i checked, and its all right.

image

## Tests Performed:

-  I went with a paladin wigher that lvl 20, so she can give us the quest.
-  Observe the gossip is correct

## How to Test the Changes:

1. .go c 32066
2. Speak with her with more that lvl 20 aprox
3. See if the gossip is correct

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1626532264671526700.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626532264671526700');
+
+-- We had 2 gossips with the id, and it was choosing the wrong one. So we insert into a new one id.
+DELETE FROM gossip_menu WHERE menuid = 61025 and textid = 10041;
+
+-- The line 10041 is the one needed for this character.
+INSERT INTO gossip_menu(menuid ,textid) VALUES (61025 ,10041);
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1626532264671526700.sql:7]: Just a little cosmetic here

**Review by LukasVolgger — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1626532264671526700.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626532264671526700');
+
+-- We had 2 gossips with the id, and it was choosing the wrong one. So we insert into a new one id.
+DELETE FROM gossip_menu WHERE menuid = 61025 and textid = 10041;
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1626532264671526700.sql:4]: Please also add backticks ` around column names

**Review by LukasVolgger — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1626532264671526700.sql (hunk 1 above)
Comment (Asthalor) [reply data/sql/updates/pending_db_world/rev_1626532264671526700.sql]: Added the backticks

File: data/sql/updates/pending_db_world/rev_1626532264671526700.sql (hunk 2 above)
Comment (Asthalor) [reply data/sql/updates/pending_db_world/rev_1626532264671526700.sql]: Added the backticks

**Comment by LukasVolgger** [comment]
Thx for the update, and pls be sure to add a new line after your code.

Ready to test!

**Review by LukasVolgger — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
Given that this is a small change and the author already proved to submit valid fixes and confirmed to have tested this, there is no reason to require further testing.

Good job @Asthalor and thanks @LukasVolgger for reviewing this

**Comment by FrancescoBorzi** [comment]
@Asthalor also, please note the small change I did to your PR description.

### Before

```
- Closes the issue https://github.com/azerothcore/azerothcore-wotlk/issues/6971
```

### After
```
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6971
- Closes https://github.com/chromiecraft/chromiecraft/issues/1212
```

this makes sure the issue(s) are closed automatically once the PR is merged to master

**Pull request merged**